### ISSUE TITLE
Specialization (tuxml.config)

### ISSUE THREAD

**Issue by FAMILIAR-project** [TuxML/ProjetIrma#95]
Based on our large-scale campaign, learning results, and domain knowledge, I suggest we pre-set the following values:
CONFIG_64BIT=y
CONFIG_DEBUG_INFO=n 
CONFIG_GCOV_PROFILE_ALL=n 
CONFIG_KASAN=n 

I guess in tuxml.config? 
Then we still use some "randomisation" (with make randconfig)

It would be nice to have thousands of such configurations in the next upcoming days

**Comment by alemasle** [comment]
I will add those new lines in the tuxml.config, do you need it right now ? I just have to repush the docker image

**Comment by FAMILIAR-project** [comment]
I think it's good to finish the current compilations and then yea update the Docker image, control that everything goes fine, and launch jobs ;)

**Comment by alemasle** [comment]
The new image has been pushed !

**Comment by FAMILIAR-project** [comment]
It is possible once the current campaign is finished to modify again the image so that the following options values are preset?

CONFIG_UBSAN_SANITIZE_ALL=n
CONFIG_RELOCATABLE=n
CONFIG_XFS_DEBUG=n 

(internal note: RANDOMIZE_BASE and X86_NEED_RELOCS will be automatically set to 'n' thanks to CONFIG_RELOCATABLE)

**Comment by alemasle** [comment]
Added !

**Comment by alemasle** [comment]
See #98

**Comment by FAMILIAR-project** [comment]
'RELOCATABLE' is sometimes "y" 

5	87805	y
22	87822	y
44	87844	y
49	87849	y
60	87860	y
61	87861	y
62	87862	y
66	87866	y
97	87897	y
118	87918	y
123	87923	y
137	87937	y
154	87954	y
161	87961	y
165	87965	y
167	87967	y
174	87974	y
185	87985	y
202	88002	y
206	88006	y
208	88008	y
218	88018	y
225	88025	y
234	88034	y

**Comment by alemasle** [comment]
What is the first number in front of the cid ?
I suspect that there is only one machine which is still running with the old tuxml.config, i have one that is running since friday. I launch it to compile during the week-end. I will check this

**Comment by FAMILIAR-project** [comment]
Still the problem (first number is "index", second number is "cid") even with recent compilations

2445  92445           y
2455  92455           y
2461  92461           y
2465  92465           y
2468  92468           y
2473  92473           y
2478  92478           y
2491  92491           y

(note: 268 configurations out of 2500 have RECLOATE to "y"; so maybe that's "old" jobs)

**Comment by alemasle** [comment]
I made a test, 12 out of 100 .config had CONFIG_RELOCATABLE=y (freshly generated), so maybe this is something rewrited by an other option ?

**Comment by FAMILIAR-project** [comment]
Strange :( 
but thanks for the test.

We really want to have "n" to all pre-defined options, including CONFIG_RELOCATABLE... 
It seems that 88 configurations are like this in your experiment. So it is clearly possible. 
On a recent CSV file, I've got 24 out of 236 having CONFIG_RELOCATABLE=y (~10% like in your experiment) 

For all 236 configurations, options values are really:
```
CONFIG_64BIT=y
CONFIG_DEBUG_INFO=n
CONFIG_GCOV_PROFILE_ALL=n
CONFIG_KASAN=n
CONFIG_UBSAN_SANITIZE_ALL=n
CONFIG_XFS_DEBUG=n
CONFIG_AIC7XXX_BUILD_FIRMWARE=n
CONFIG_AIC79XX_BUILD_FIRMWARE=n
CONFIG_WANXL_BUILD_FIRMWARE=n
```

so it seems a very strange problem :(

**Comment by alemasle** [comment]
From where could it comes?

**Comment by FAMILIAR-project** [comment]
I close this issue. I am opening a new, proper, related issue (specialization fails)
